desktop_version/src/Script.cpp

```diff
@@ -2114,25 +1798,28 @@ void scriptclass::run(void)
 			}
 			else if (words[0] == "createlastrescued")
 			{
-				if (game.lastsaved==2)
-				{
-					r=14;
-				}
-				else if (game.lastsaved==3)
-				{
-					r=15;
-				}
-				else if (game.lastsaved==4)
-				{
-					r=13;
-				}
-				else if (game.lastsaved==5)
-				{
-					r=16;
-				}
-				else
+				int crew_color = 1;
```

Comment:
Why does this default to 1? Also, it doesn't seem like `crew_color` is used here. And I'm wondering why the `r` assignments don't just use the color enums.

Reply:
I've replaced that entire section with `graphics.crewcolour()`, which I seem to have forgotten was a thing. Whoops.